src/analyze/analyze-critical-chain.c

```diff
@@ -3,7 +3,7 @@
 #include "analyze-critical-chain.h"
 #include "analyze-time-data.h"
 #include "analyze.h"
-#include "bus-error.h"
+#include "bus-error-util.h"
```

Comment:
I'd like that this renaming should be done in a different commit, as this commit is one of the most important commits in this PR. Making this commit smaller is better for reviewing this PR.

Reply:
Actually, it's not a rename. I'm splitting `bus-error.h` into two files: `bus-error.h` and `bus-error-util.h`. The include is changed because the files outside of `src/libsystemd/` need to include the new header. It wouldn't make sense to make this split without the other changes.